src/main/kotlin/com/atlassian/performance/tools/jirasoftwareactions/api/page/BrowseBoardsPage.kt

```diff
@@ -36,15 +36,15 @@ class BrowseBoardsPage(
             return emptyList()
         }
         driver.wait(
-            Duration.ofSeconds(5), 
-            elementToBeClickable(By.cssSelector("#ghx-manage-boards-filter a"))
+            Duration.ofSeconds(5),
+            elementToBeClickable(By.xpath("//*[@aria-controls='board-types']"))
```

Comment:
XPath is useful for complex hierarchical locators. Here a simple CSS selector is sufficient:
```suggestion
            elementToBeClickable(By.cssSelector("[aria-controls='board-types']"))
```